src/altinn-app-frontend/src/shared/resources/dataLists/dataListsSlice.ts

```diff
@@ -0,0 +1,93 @@
+import { fetchDataListsSaga } from 'src/shared/resources/dataLists/fetchDataListsSaga';
+import { createSagaSlice } from 'src/shared/resources/utils/sagaSlice';
+import type {
+  IDataListsState,
+  IFetchDataListsFulfilledAction,
+  IFetchDataListsRejectedAction,
+  IFetchingDataListsAction,
+  ISetDataLists,
+  ISetDataListsPageNumber,
+  ISetDataListsPageSize,
+  ISetDataListsWithIndexIndicators,
+  ISetSort,
+} from 'src/shared/resources/dataLists';
+import type { MkActionType } from 'src/shared/resources/utils/sagaSlice';
+
+const initialState: IDataListsState = {
+  dataLists: {},
+  dataListsWithIndexIndicator: [],
+  error: null,
```

Comment:
Are you using the error anywhere? See `src/selectors/getErrors.ts`, you should probably add your error state there as well. It's not really optimal either, but at least we react to an error when fetching the list data instead of rejecting and silently ignoring the problem.